src/models/Share.js

```diff
@@ -0,0 +1,29 @@
+import { Schema, model } from 'mongoose';
+
+const ShareSchema = new Schema({
+  plateforme: {
+    type: String,
+  },
+  user: {
+    type: Schema.Types.ObjectId,
+    ref: 'User',
+  },
+  post: {
```

Comment:
@gracelungu Job and Post reference should be combined into one and have another property to differentiate them

Reply:
I did that because I wanted to put the reference for both Posts and Jobs